content/rc/cloud-integrations/pulumi/_index.md

```diff
@@ -0,0 +1,99 @@
+
+Pulumi Redis Cloud Provider `resources` include :
+
+* `Subscription`: The basic building block of a Redis Cloud subscription.
+* `SubscriptionDatabase` : Represents a Redis database which belongs to a specific Redis Cloud subscription.
+* `SubscriptionPeering` : A VPC peering connection (AWS or GCP) to a specific Redis Cloud subscription.
+* `CloudAccount` : Represents an AWS account in which you want to deploy Redis Cloud infrastructure components.
+
+{{<note>}}
+The "bring your own AWS account" for Redis Cloud has now been deprecated. The "CloudAccount" resource is only made available for legacy Redis Cloud integrations.
+{{</note>}}
+
+* `ActiveActiveSubscription`: The basic building block of an active-active Redis Cloud subscription.
+* `ActiveActiveSubscriptionDatabase`: Represents a Redis database which belongs to a specific Redis Cloud active-active subscription.
+* `ActiveActiveSubscriptionRegions`: The different regions where the active-active subscription will be deployed.
+* `ActiveActiveSubscriptionPeering`: A VPC peering connection (AWS or GCP) to a specific Redis Cloud active-active subscription.
+
+Pulumi Redis Cloud Provider `functions` include :
+
+* `GetCloudAccount`: Get the information related to the AWS account.
+
+{{<note>}}
+The "bring your own AWS account" for Redis Cloud has now been deprecated. The "CloudAccount" resource is only made available for legacy Redis Cloud integrations.
+{{</note>}}
+
+* `GetDataPersistence`: Get the type of database persistence.
+* `GetDatabase`: Get the information related to a specific database.
+* `GetDatabaseModules`: Get the modules for a specific database.
+* `GetPaymentMethod`: Get the payment method related to the Redis Cloud account.
+* `GetRegions`: Get the regions related to an active-active 
+* `GetSubscription`: Get the information related to a specific subscription.
+* `GetSubscriptionPeerings`: Get the VPC peerings (AWS or GCP) related to a specific subscription.
+
+{{<note>}}
+For more information on the different Redis Cloud Provider Resources and Functions, please refer to Pulumi's [official documentation for the Redis Cloud provider](https://www.pulumi.com/registry/packages/rediscloud/api-docs/).
+{{</note>}}
+
+## Pulumi Quick-Start Python Project
+
+Let's examine an example of creating a "Flexible" Redis Cloud subscription using Python.
+
+1.  Make sure you have your [Pulumi environment installed](https://www.pulumi.com/docs/install/).
+
+1.  In your Python project, create an empty folder and from this folder run `pulumi new rediscloud-python`.
+
+1.  Next, enter a project name, description, and stack name.
+
+1.  The next step will ask for your Redis Cloud access and secret keys. Since Pulumi uses the Redis Cloud API in the background, an **access Key** and **secret key** are required. For more information on creating these keys, please refer to [Cloud API Keys]({{< relref "/rc/api/get-started/manage-api-keys.md" >}})
```

Comment:
```suggestion
1.  Enter your Redis Cloud access and secret keys. 

    Since Pulumi uses the Redis Cloud API in the background, an **access key** and **secret key** are required. For more information on creating these keys, please refer to [Cloud API Keys]({{< relref "/rc/api/get-started/manage-api-keys.md" >}})
```